### DISCUSSION TITLE
Twitter Timeline ne s'affiche pas

### DISCUSSION THREAD

**Discussion by TibowWebdesign** [AmauriC/tarteaucitron.js#1063] [Q&A]
Hello,

Ma Twitter Timeline ne s'affiche pas même lorsque j'accepte les cookies Twitter Timeline.
![twitter-timeline-tac-hs](https://user-images.githubusercontent.com/28344592/217217015-038e5c96-0de2-41f8-b43d-6f6600089845.jpg)
La timeline semble  pourtant exister quand on regarde l'inspecteur de code Chrome...

Le code TAC utilisé :
<span class="tacTwitterTimelines"></span><a class="twitter-timeline" href="https://twitter.com/AnaeeFrance?ref_src=twsrc%5Etfw" data-tweet-limit="8" data-dnt="true" data-width="355" data-height="950" data-theme="light" data-link-color="#378d3a"></a>

Je ne vois pas ce qui pêche :(

Merci par avance

**Comment by AmauriC** [comment]
Salut,

En effet ça semble chargé. Possible d'avoir l'url de la page?

**Reply by TibowWebdesign** [reply]
https://anaeefrance.thibautsoufflet.fr/

**Reply by AmauriC** [reply]
De mon côté ça fonctionne bien. Il n'y a pas un adblocker par exemple qui empêche le chargement côté navigateur?

![Capture d’écran 2023-02-07 à 11 23 43](https://user-images.githubusercontent.com/8764074/217218975-98f052b4-4986-4246-afee-eb427ea28498.png)

**Reply by TibowWebdesign** [reply]
Exact j'ai ces messages dans la console, possible de contourner le AddBlock ?

GET https://platform.twitter.com/_next/static/chunks/runtime-5988b8a698c47e6a5fbc.js net::ERR_BLOCKED_BY_CLIENT
AnaeeFrance:1          GET https://platform.twitter.com/_next/static/chunks/main-fd9ef5eb169057cda26d.js net::ERR_BLOCKED_BY_CLIENT
AnaeeFrance:1          GET https://platform.twitter.com/_next/static/chunks/pages/_app-de607716f6c37dff4b26.js net::ERR_BLOCKED_BY_CLIENT
AnaeeFrance:1          GET https://platform.twitter.com/_next/static/chunks/pages/timeline-profile/screen-name/%5BscreenName%5D-c33f0b02841cffc3e9b4.js net::ERR_BLOCKED_BY_CLIENT
AnaeeFrance:1          GET https://platform.twitter.com/_next/static/chunks/modules.20f98d7498a59035a762.js net::ERR_BLOCKED_BY_CLIENT
AnaeeFrance:1          GET https://platform.twitter.com/_next/static/KDYLOkGzYwcqxkInYxTHF/_buildManifest.js net::ERR_BLOCKED_BY_CLIENT
AnaeeFrance:1          GET https://platform.twitter.com/_next/static/KDYLOkGzYwcqxkInYxTHF/_ssgManifest.js net::ERR_BLOCKED_BY_CLIENT

**Comment by AmauriC** [comment]
ERR_BLOCKED_BY_CLIENT = chargement de la requête bloqué par une extension.

**Reply by TibowWebdesign** [reply]
Merci, visiblement c'est mon extension Chrome DuckDuckGo qui créée le blocage.
👍
